app/assets/stylesheets/common/components/topic-filter-education.scss

```diff
@@ -0,0 +1,27 @@
+@use "lib/viewport";
+
+.empty-topic-filter-education {
```

Comment:
Could we shorten this parent class name? What's "education" doing in there? I'd rather have something like `empty-state` and if we must differentiate use modifiers like `empty_state --topic-filer`. That way the classnames of child items remains more concise.